src/sentry/utils/data_filters.py

```diff
@@ -26,6 +26,7 @@ class FilterStatKeys(object):
     WEB_CRAWLER = 'web-crawlers'
```

Comment:
Just realized the left side is singular and the right side is plural and my OCD is firing on all cylinders

Reply:
:samsung: